### ISSUE TITLE
kendo.support.scrollbar() reports zero in Mac browsers in some scenarios

### ISSUE THREAD

**Issue by dimodi** [telerik/kendo-ui-core#894]
Ticket ID 946524

**Comment by dimodi** [comment]
Works here: http://dojo.telerik.com/@dimodi/EDEWO

Does not work on Kendo UI demo site.

**Comment by underlog** [comment]
Unless I am missing something, the behaviour you refer to is by design and should return zero - `0`. By default OS X uses **scroll hints**. 
<img width="780" alt="screen shot 2015-07-19 at 2 32 20 pm" src="https://cloud.githubusercontent.com/assets/13347/8765690/71d2382e-2e22-11e5-8455-410413a238e5.png">

**Comment by dimodi** [comment]
Hi Petyo, thanks for stepping in and for the information.

I tried to dig into this issue again, but unfortunately, I am no longer able to reproduce it. `kendo.support.scrollbar()` consistently returns 0 or 15, depending on the setting shown on the above screenshot. Previously, it returned 0 on our demo site and 15 in an isolated Dojo example (without fiddling with the scrollbar setting, which I was unaware of), so it looked like the demo site's styles or scripts were involved in some way. If you remember, we even checked two Grid demos on your machine and one of them lacked a Grid scrollbar, while the other had it (the demos were *"Basic usage"* and *"Virtualization of remote data"*, if I remember correctly).

Judging by ticket 946524 and my workaround, the scrollbar was rendered by the browser, but the virtual scrollbar container `<div>` had a zero width, which was incorrect. Normally, the width of this `<div>` is set, based on the value returned by `kendo.support.scrollbar()`.

I just found out that our Mac test machine was recently preinstalled, which may be related.

I am closing this issue for now.
